doc/source/user_guide/groupby.rst

```diff
@@ -205,7 +205,7 @@ By default the group keys are sorted during the ``groupby`` operation. You may h
    df2.groupby(["X"], sort=False).sum()
 
 
-Note that ``groupby`` will preserve the order in which *observations* are sorted *within* each group.
+Note that ``groupby`` will preserve the order in which *observations* are sorted *within* each group .
```

Comment:
why make this change?

Reply:
In the comments of the [issue](https://github.com/pandas-dev/pandas/issues/47529) it was mentioned to add also a description in the "User Guide".

Reply:
@raanasn - it looks like you're adding an unnecessary space before the period here.

Reply:
exactly - if you revert this the rest look good to me

Reply:
Typo, thanks!!